example/lib/main.dart

```diff
@@ -51,6 +51,19 @@ class _ExampleBrowser extends State<ExampleBrowser> {
       await _controller.setPopupWindowPolicy(WebviewPopupWindowPolicy.deny);
       await _controller.loadUrl('https://flutter.dev');
 
+      print(await _controller.executeScript('document.activeElement.getBoundingClientRect()')); // ❌ return null

```

Comment:
Please migrate all prints to proper tests.

Reply:
Sorry buy i'm have no time to move those examples to test cases, so i removed them.